modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/storage_formatter.js

```diff
@@ -95,4 +17,4 @@ AUI.add(
 	{
 		requires: ['aui-base', 'datatype-number-format']
```

Comment:
We can certainly remove 'datatype-number-format' requirement, probably 'aui-base' as well.